samples/client/petstore/rust-reqwest/src/models/api_response.rs

```diff
@@ -0,0 +1,90 @@
+/* 
+ * OpenAPI Petstore
+ *
+ * This is a sample server Petstore server. For this sample, you can use the api key `special-key` to test the authorization filters.
+ *
+ * OpenAPI spec version: 1.0.0
+ * 
+ * Generated by: https://openapi-generator.tech
+ */
+
+/// ApiResponse : Describes the result of uploading an image resource
+
+#[allow(unused_imports)]
+use serde_json::Value;
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct ApiResponse {
+  #[serde(rename = "code")]
+  code: Option<i32>,
+  #[serde(rename = "type")]
+  _type: Option<String>,
+  #[serde(rename = "message")]
+  message: Option<String>
+}
+
+impl ApiResponse {
+  /// Describes the result of uploading an image resource
+  pub fn new() -> ApiResponse {
+    ApiResponse {
+      code: None,
+      _type: None,
+      message: None
+    }
+  }
+
+  pub fn set_code(&mut self, code: i32) {
+    self.code = Some(code);
+  }
+
+  pub fn with_code(mut self, code: i32) -> ApiResponse {
+    self.code = Some(code);
+    self
+  }
```

Comment:
What is the purpose of all these getters and setters? Why not have the fields public?

Reply:
Copied from original code. Don't know the reason...

But I imagine the purpose is to get a "fluent API", with method calls chained: `model.with_code("code").with_name("name")...`. It is a common pattern in other languages like `java` to keep fields private and to provide this access methods.

Reply:
I note for later, but I won't correct it in this branch: it would be a breaking change in generated API since models are shared between hyper and reqwest versions.

Reply:
I note this for later, but I won't correct it in this branch: it would be a breaking change in generated API since models are shared between hyper and reqwest versions.